### PR TITLE
updated the environment docs with the database and extension

### PR DESCRIPTION
**Pull request by harshad16**
updated the environment docs with the database and extension
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Related-to: 
https://github.com/thoth-station/thoth-application/issues/726
https://github.com/thoth-station/thoth-application/issues/725

## Does this require new deployment ?

Not needed

## Description

document update with information on how to add extension to database.

### CODE REVIEW

File: docs/environments.md [hunk 1]
```diff
@@ -1,7 +1,61 @@
...
 
-This section describes the different environments used to deploy all the Thoth Services to. All environments should be deployed using ArgoCD.
+This section describes the different environments used to deploy all the Thoth Services to OpenShift Cluster.  
 
+*Note*: All environments can be deployed using ArgoCD or Kustomize.
+
+
+## Database Setup
+
+Project thoth uses Postgresql database for knowlegde graph storage.
```
Comment (pacospace) [inline docs/environments.md:10]: `Thoth`

File: docs/environments.md [hunk 2]
```diff
@@ -1,7 +1,61 @@
...
+
+- Deployment manifest can be found at: [Graph Manifest](../core/overlays/cnv-prod/graph-prod)
+- we manage the storage via different project thoth components which utlises the [thoth-storage](https://pypi.org/project/thoth-storages/) module. [Details](https://github.com/thoth-station/storages#thoth-storages)
+- Local setup and management information can be acquired at [thoth-storages](https://github.com/thoth-station/storages#thoth-storages)
+
+Extensions:
+
+**amcheck**: we have setup amcheck extension for detecting database corruption errors.
+The [Issue](https://github.com/thoth-station/storages/issues/1586) based on which it was decided to use this extensions.
```
Comment (pacospace) [inline docs/environments.md:19]: `extension`

File: docs/environments.md [hunk 3]
```diff
@@ -1,7 +1,61 @@
...
+
+The check for database corruption is executed from metrics-exporter.  
+The setup of the extension is to be performed with setup of database. init-job would help if being used, if manual process is to be used then perform the following action:
+
+Execute `CREATE EXTENSION amcheck;` in the database where amcheck is to be set. we rsh into the pod handling the postgresl and perform following action
```
Comment (pacospace) [inline docs/environments.md:24]: `We`

File: docs/environments.md [hunk 4]
```diff
@@ -1,7 +1,61 @@
...
 
+*Note*: All environments can be deployed using ArgoCD or Kustomize.
+
+
+## Database Setup
+
+Project thoth uses Postgresql database for knowlegde graph storage.
+
+- Deployment manifest can be found at: [Graph Manifest](../core/overlays/cnv-prod/graph-prod)
+- we manage the storage via different project thoth components which utlises the [thoth-storage](https://pypi.org/project/thoth-storages/) module. [Details](https://github.com/thoth-station/storages#thoth-storages)
```
Comment (pacospace) [inline docs/environments.md:13]: `Thoth`

File: docs/environments.md (hunk 3 above)
Comment (pacospace) [inline docs/environments.md:24]: `:` at the end

File: docs/environments.md (hunk 1 above)
Comment (harshad16) [reply docs/environments.md]: thanks

File: docs/environments.md (hunk 3 above)
Comment (harshad16) [reply docs/environments.md]: Thanks :)

Comment (harshad16) [reply docs/environments.md]: Thanks :)

File: docs/environments.md (hunk 4 above)
Comment (harshad16) [reply docs/environments.md]: Thanks :)

File: docs/environments.md (hunk 2 above)
Comment (harshad16) [reply docs/environments.md]: Thanks :)

**Comment by harshad16** [comment]
Ready for review again.

**Review by pacospace — COMMENTED** [review]
lgtm! 💯 👏🏼

File: docs/environments.md [hunk 5]
```diff
@@ -1,6 +1,60 @@
...
+```
+
+It can be verified if installed via `\dx`
+
+```bash
+thoth=# \dx
+                                    List of installed extensions
+        Name        | Version |   Schema   |                        Description
+--------------------+---------+------------+-----------------------------------------------------------
+amcheck            | 1.0     | public     | functions for verifying relation integrity
```
Comment (goern) [inline docs/environments.md:44]: is this somethign we have declared in some yaml? or can put to a file in a repo?

Comment (harshad16) [reply docs/environments.md:44]: we can set this from a component that talks to storage.
so we will have this in init-job, so some newly setup their environment can benefit from it.

**Pull request merged**